pkg/lib/console.css

```diff
@@ -57,6 +58,6 @@
 }
 
 /* Ensure the console fits to its container (and doesn't attempt to go beyond the limits) */
-.xterm-screen, .xterm-viewport {
```

Comment:
@garrett why you dropped `.xterm-screen` from here?

Reply:
It didn't seem needed?